crowbar_framework/app/models/node_object.rb

```diff
@@ -1207,7 +1207,7 @@ def run_ssh_cmd(cmd)
     args = ["sudo", "-i", "-u", "root", "--", "timeout", "-k", "5s", "15s",
             "ssh", "-o", "ConnectTimeout=10",
             "root@#{@node.name}",
-            cmd.to_s
+            "\"#{cmd}\""
```

Comment:
6d3014a seems to no longer exist.  If this function cannot correctly handle commands containing double-quotes characters, then even if none of the commands currently contain double-quote characters, Murphy's Law dictates that they will soon ;-)

BTW why does it have to sudo in order to ssh?  I guess only `root` has the required authorized key?